File: week_1_C/pset_1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //ask for height
    int height;
    do
    {
        height = get_int("height: ");
    }
    while (height < 1 || height > 8);

    //first loop for printing rows
    for (int i = 0; i < height; i++)
    {
        //second loop for printing spaces
        for (int k = 1; k < height - i; k++)
        {
            printf(" ");
        }

        //third loop for prinitng hashes
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week_6_Python/pset_6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    try:
        height = get_int("Height: ")
    except ValueError:
        continue
    if 1 <= height <= 8:
        break

for i in range(height):
    for j in range(height - (i + 1)):
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")
    print("  ", end="")
    for k in range(i + 1):
        print("#", end="")
    print()